### PR TITLE
Preparation for Tables patch release for August 2021

### PR DESCRIPTION
**Pull request by vcolin7**

### CODE REVIEW

File: sdk/tables/azure-data-tables/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
 
-## 12.2.0-beta.1 (Unreleased)
+## 12.1.1 (2021-08-12)
 
+### Bugs Fixed
+- Fixed an issue that made getting entities from a Cosmos endpoint fail as it does not return a `Timestamp@odata.type` property in a `TableEntity` alongside the `Timestamp` property, like Storage endpoint do. This is apparently intended behavior, so we now always convert `Timestamp` to `OffsetDateTime` as it is a reserved property name and will always be provided by the service.
+- Updated clients to properly map an internal HTTP exception to the public type `TableServiceException` in operations such as `getAccessPolicies()`, `getProperties()`, `setProperties()` and `getStatistics()`, including their `withResponse` variants.
+- Fixed batch operations to properly log exceptions other than `TableTransactionFailedException`.
+
+### Other Changes
```
Comment (srnagar) [inline sdk/tables/azure-data-tables/CHANGELOG.md:10]: Remove this empty section

**Review by srnagar — APPROVED** [review]

File: sdk/tables/azure-data-tables/CHANGELOG.md (hunk 1 above)
Comment (vcolin7) [reply sdk/tables/azure-data-tables/CHANGELOG.md:10]: It's a parent header to `Dependency Updates`.

Comment (alzimmermsft) [reply sdk/tables/azure-data-tables/CHANGELOG.md:10]: Is this a new CHANGELOG pattern we are meant to follow?

**Review by alzimmermsft — APPROVED** [review]

File: sdk/tables/azure-data-tables/CHANGELOG.md (hunk 1 above)
Comment (g2vinay) [reply sdk/tables/azure-data-tables/CHANGELOG.md:10]: yes, Other Changes is a new section that was added recently.
Anything under here, will be ignored by eng sys automation.
So dependency updates are fine to be added here.

**Review by g2vinay — APPROVED** [review]

**Pull request merged**